Tests/AppTests/__Snapshots__/SearchShowModelTests/test_SearchShow_Model_canonicalURLAllowList.1.html

```diff
@@ -0,0 +1 @@
e" type="application/rss+xml" title="Swift Package Index – Recent Major Releases" href="http://localhost:8080/releases.rss?major=true"/><link rel="alternate" type="application/rss+xml" title="Swift Package Index – Recent Major & Minor Releases" href="http://localhost:8080/releases.rss?major=true&minor=true"/><link rel="alternate" type="application/rss+xml" title="Swift Package Index – Recent Pre-Releases" href="http://localhost:8080/releases.rss?pre=true"/><script src="/main.js?test" data-turbolinks-track="reload" defer></script></head><body><div class="staging">Staging / Development</div><header><div class="inner"><a href="/"><h1><img alt="The Swift Package Index logo." src="/images/logo.svg"/>Swift Package Index</h1></a><nav><ul><li><a class="supporters" href="/supporters">Supporters</a></li><li><a href="/add-a-package">Add a Package</a></li><li><a href="https://blog.swiftpackageindex.com">Blog</a></li><li><a href="/faq">FAQ</a></li></ul></nav></div></header><nav class="breadcrumbs"><div class="inner"><ul><li><a href="/"><span>Home</span></a></li><li><span>Search Results for &ldquo;alamo&rdquo;</span></li></ul></div></nav><main><div class="inner"><section class="search"><form action="/search"><input id="query" name="query" type="search" placeholder="Search" spellcheck="false" autocomplete="off" data-gramm="false" data-focus="true" value="alamo"/><button type="submit"><div title="Search"></div></button></form><div class="filter-suggestions">Add filters to narrow search results. <span class="learn_more"><a href="/faq#search-filters">Learn more</a>.</span></div></section><section class="search-results"><p>Results for &ldquo;<strong>alamo</strong>&rdquo;&hellip;</p><section class="package-results"><h4>Matching packages</h4><p>No packages found.</p><ul id="package-list" data-focus-query-field="true"></ul><ul class="pagination"><li class="previous"><a href="/search?query=alamo&page=1">Previous Page</a></li></ul></section></section></div></main><footer><div class="inner"><nav><ul><li><a href="https://blog.swiftpackageindex.com">Blog</a></li><li><a href="https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server">GitHub</a></li><li><a href="/privacy">Privacy and Cookies</a></li><li><a href="https://swiftpackageindex.statuspage.io">Uptime and System Status</a></li><li><a href="/build-monitor">Build System Monitor</a></li><li><a href="https://twitter.com/swiftpackages">Twitter</a></li></ul><small>The Swift Package Index is entirely funded by community sponsorship. Thank you to <a href="https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server#funding-and-sponsorship">all our sponsors for their generosity</a>.</small><small>The Swift Package Index is operated by SPI Operations Limited, a company registered in the UK with company number 13466692.</small><a rel="me" href="https://mas.to/@SwiftPackageIndex"></a><a rel="me" href="https://mas.to/@SwiftPackageUpdates"></a></nav></div></footer><div class="staging">Staging / Development</div></body></html>
```

Comment:
This snapshot is less than ideal given the compressed HTML we output, but again it seems reasonable to me.

Reply:
Oh, I hadn't even seen this comment yet - this should be fixed by my suggestion above (and could even be avoid entirely by inspecting the `HTML` node object).